code/mergeblock/merge.py

```diff
@@ -1,18 +1,17 @@
 #Jakub Podolak for Plastic Monkeys Cansat 2019
 
-import subprocess
 import configparser
 import msgpack
 import time
 import sys
 import re
+import logging
 
 #function to read paths to sensors' logs from config file
 def get_directories(config):
-    directories = []
     paths = config.items("PATHS")
-    for key, path in paths:
-        directories.append(path)
+    print(paths)
```

Comment:
I Know that this line is not required here, i will remove that after your review